dashboards-observability/public/components/explorer/docTable/docViewRow.tsx

```diff
@@ -36,6 +35,12 @@ export const DocViewRow = (props: IDocViewRowProps) => {
   const [detailsOpen, setDetailsOpen] = useState<boolean>(false);
   const [surroundingEventsOpen, setSurroundingEventsOpen] = useState<boolean>(false);
   const [openTraces, setOpenTraces] = useState<boolean>(false);
+  const [flyoutToggleSize, setFlyoutToggleSize] = useState(true);
+  const [logTraceId, setLogTraceId] = useState('');
+
+  const isValidTraceId = (traceId: string) => {
```

Comment:
why redefine it here? also why 32? i don't think it's fixed

Reply:
Thanks for the catch. The 32 is declared from https://www.w3.org/TR/trace-context/#trace-id, got the link from open-telemetry docs.

Reply:
@joshuali925 If it seems unnecessary, I can remove it.

Reply:
should be ok. I was thinking about shorter trace IDs but they will be padded with leading zeros in data-prepper